src/Symfony/Component/Serializer/Tests/Normalizer/UidNormalizerTest.php

```diff
@@ -146,14 +146,9 @@ public function testSupportsDenormalizationForNonUid()
         $this->assertFalse($this->normalizer->supportsDenormalization('foo', \stdClass::class));
     }
 
-    /**
-     * @group legacy
-     */
     public function testSupportOurAbstractUid()
     {
-        $this->expectDeprecation('Since symfony/serializer 6.1: Supporting denormalization for the "Symfony\Component\Uid\AbstractUid" type in "Symfony\Component\Serializer\Normalizer\UidNormalizer" is deprecated, use one of "Symfony\Component\Uid\AbstractUid" child class instead.');
```

Comment:
Note that the `ExpectDeprecationTrait` can be removed when `expectDeprecation` is not called.